02-peer-protocol.md

```diff
@@ -70,6 +70,9 @@ If this fails at any stage, or a node decides that the channel terms
 offered by the other node are not suitable, the channel establishment
 fails.
 
+Note that multiple channels can operate in parallel, but only after
```

Comment:
Concurrent funding from both sides should Just Work.  But we would need to re-add temp channel ids (easy now we have 34 bytes, though, we can just use random value).

Hmm, OK, you convinced me :)  I didn't add a limit for now, but see latest push.

I've thought about forgetting channel if we disconnect before receipt of funding_signed (or send of funding_signed for the accepter).  The corner case is that B has sent funding_signed and A didn't get it.  B retransmits, and then it gets an error, which I think is works fine.  I'll create a separate patch for this once we're done here (I've filed #120 for this).